mono/eglib/gstr.c

```diff
@@ -914,22 +914,17 @@ g_utf16_asciiz_equal (const gunichar2 *utf16, const char *ascii)
 	}
 }
 
-gchar *
-g_strdelimit (gchar *string, const gchar *delimiters, gchar new_delimiter)
+void
```

Comment:
There is now no way to verify if the call succeeded. What's the point of this change?

Reply:
Nobody checks the return value, and it is only ever the parameter returned back to the caller unchanged, and it isn't really failure, it is just don't replace what isn't present, and what caller doesn't assume is present -- doesn't seem worthwhile.

Reply:
@luhenry, I agree with @jaykrell  - I don't think a non-`void` return type serves any purpose here.

Reply:
On one hand the return indeed doesn't serve a purpose right now, but since it's actually used to return if there was an error or not, it could be used by callers.